### DISCUSSION TITLE
DSIF and RRQP files sets

### DISCUSSION THREAD

**Discussion by creinemann** [JVital2013/vitality-goes#43] [General]
[GOES20230705154049.zip](https://github.com/JVital2013/vitality-goes/files/11961721/GOES20230705154049.zip)

**Comment by JVital2013** [comment]
You already know this, but I'm posting this here for completion:

**RRQPE:**  Requires a gradient fix in goesproc-goesr.conf, which I just pushed to the Vitality GOES repo. Easy fix.

**DSIs**
- These can be decoded with the patched goestools from [this PR](https://github.com/pietern/goestools/pull/163), and colored with the PRs [here](https://github.com/creinemann/EMWIN-NON-CMIP-SCRIPTS-FOR-GOESTOOLS/pull/2) and [here](https://github.com/creinemann/EMWIN-NON-CMIP-SCRIPTS-FOR-GOESTOOLS/pull/3). 
- I'm conflicted if I'm going to include sample configs for this with Vitality GOES, since the patch has not been merged back into the official goestools repo

If I had to guess, very few people are running the DSI patch now. Of course, Vitality GOES users with the patch can simply update their own l2.ini file to handle the new files.
